src/cargo/ops/cargo_package.rs

```diff
@@ -49,7 +49,7 @@ pub fn package(ws: &Workspace,
         }).collect();
         list.sort();
         for file in list.iter() {
-            println!("{}", file.display());
+            eprintln!("{}", file.display());
```

Comment:
This is machine-consumable, and is the only way to get the list of files that would be packaged AFAIK.

Reply:
I concur. However, this PR isn’t so much about distinguishing machine-readable output from human-readable output across Cargo. It is rather about distinguishing _standardized_, _exclusively_ machine-readable ouput from other output. But can’t we make principled decision about what ‘machine-readable’ means within Cargo. If that means making all such output be JSON, then let’s do that rather than making local exceptions left and right.

Reply:
The change as-is will break code that calls this command and expects to see something on stdout.